ginkgo: Get rid of K8sUpdates

brb opened the pull request:
It got replaced by ci-e2e-upgrade.

brb committed:
ginkgo: Get rid of K8sUpdates

It got replaced by ci-e2e-upgrade.

Signed-off-by: Martynas Pumputis <m@lambda.lt>

ldelossa reviewed (approved):
LGTM

gandro commented on .github/workflows/conformance-k8s-kind.yaml:
```diff
@@ -196,6 +196,16 @@ jobs:
+          for p in $(pidof cilium-agent); do
+            num=$(sudo lsof -p $p | wc -l)
+            if [ "$num" -gt "500" ]; then
+              echo "cilium-agent($p) has more than 500 file descriptors (potential leak)"
```
Apologies for the drive-by comment  - but if implementable I think this would be a good Prometheus metric to have (which we could then also assert in `cilium connectivity check` in all workflows)

brb replied:
I moved the commits to https://github.com/cilium/cilium/pull/36241.

brb committed:
test/runtime: Start Cilium before running FQDN tests

Otherwise, the tests fail to start if they run after net_policies.go.

Suggested-by: Harsimran Pabla <hpabla@isovalent.com>
Signed-off-by: Martynas Pumputis <m@lambda.lt>

brb committed:
test/runtime: Start Cilium before monitor tests

Same reasoning as in the previous commit.

Signed-off-by: Martynas Pumputis <m@lambda.lt>
